src/sentry/monitors/utils.py

```diff
@@ -18,3 +21,51 @@ def signal_first_monitor_created(project: Project, user, from_upsert: bool):
         first_cron_monitor_created.send_robust(
             project=project, user=user, from_upsert=from_upsert, sender=Project
         )
+
+
+def get_alert_rule(project: Project, user: User, monitor: Monitor, alert_rule: dict):
+    target_identifier = alert_rule.get("targetIdentifier", user.id)
+    target_type = alert_rule.get("targetType", "Member")
```

Comment:
key values from frontend here

Reply:
This feels a little to deep to be dealing with frontend data, let's use snake case for these and we can do the translation in the endpoint.

I wonder if we could even make use of the `CamelSnakeSerializer` somewhere